frontend-admin/src/app/views/organization/add-app-modal/add-app-modal.html

```diff
@@ -0,0 +1,47 @@
+<div id="add-app-modal">
+  <!-- Modal Header -->
+  <div class="modal-header">
+    <h3 class="modal-title">Add more apps</h3>
+    <div class="close" type="button" ng-click="vm.closeModal()" >×</div>
+  </div>
+  <!-- Modal Body -->
+  <div class="modal-body" style="max-height:{{vm.modal_height}};">
+    <div class="alert alert-danger modal-error" ng-show="vm.displayError">
```

Comment:
Prefer to use a `ng-if` as  it will not be displayed to the user most of the time.